sympy/integrals/transforms.py

```diff
@@ -1137,6 +1160,541 @@ def sbs(expr):
+        if ma2:
+            debug('_laplace_apply_rules match:')
+            debug('      f:    %s ( %s, %s )'%(f, ma1, ma2))
+            debug('      rule: multiply with sinh (1.5)')
+            L = _laplace_apply_rules(ma1[z], t, s, **hints)
+            try:
+                r, p, c = L
+                return ((r.subs(s, s-ma2[a])-r.subs(s, s+ma2[a])).simplify()/2,
+                        p+Abs(ma2[a]), c)
+            except TypeError:
+                return (L.subs(s, s-ma2[a])-L.subs(s, s+ma2[a]))/2
+    ma1 = func.match(cosh(y)*z)
+    if ma1:
+        ma2 = ma1[y].collect(t).match(a*t)
+        if ma2:
+            debug('_laplace_apply_rules match:')
+            debug('      f:    %s ( %s, %s )'%(f, ma1, ma2))
+            debug('      rule: multiply with cosh (1.6)')
+            L = _laplace_apply_rules(ma1[z], t, s, **hints)
+            try:
+                r, p, c = L
+                return ((r.subs(s, s-ma2[a])+r.subs(s, s+ma2[a])).simplify()/2,
+                        p+Abs(ma2[a]), c)
+            except TypeError:
+                return (L.subs(s, s-ma2[a])+L.subs(s, s+ma2[a]))/2
+    ma1 = func.match(sin(y)*z)
+    if ma1:
+        ma2 = ma1[y].collect(t).match(a*t)
+        if ma2:
+            debug('_laplace_apply_rules match:')
+            debug('      f:    %s ( %s, %s )'%(f, ma1, ma2))
+            debug('      rule: multiply with sin (1.7)')
+            L = _laplace_apply_rules(ma1[z], t, s, **hints)
+            try:
+                r, p, c = L
+                return (-I*(r.subs(s, s-I*ma2[a])-r.subs(s, s+I*ma2[a])).simplify()/2, p, c)
+            except TypeError:
+                return -I*(L.subs(s, s-I*ma2[a])-L.subs(s, s+I*ma2[a]))/2
+    ma1 = func.match(cos(y)*z)
+    if ma1:
+        ma2 = ma1[y].collect(t).match(a*t)
+        if ma2:
+            debug('_laplace_apply_rules match:')
+            debug('      f:    %s ( %s, %s )'%(f, ma1, ma2))
+            debug('      rule: multiply with sin (1.7)')
+            L = _laplace_apply_rules(ma1[z], t, s, **hints)
+            try:
+                r, p, c = L
+                return ((r.subs(s, s-I*ma2[a])+r.subs(s, s+I*ma2[a])).simplify()/2, p, c)
+            except TypeError:
+                return (L.subs(s, s-I*ma2[a])+L.subs(s, s+I*ma2[a]))/2
+    ma1 = f.match(a*Derivative(g, (t, n)))
+    if ma1 and ma1[g].args[0] == t and ma1[n].is_integer:
+        debug('_laplace_apply_rules match:')
+        debug('      f:    %s'%(f,))
+        debug('      rule: time derivative (1.11, 1.12)')
+        d = []
+        for k in range(ma1[n]):
+            if k==0:
+                y = ma1[g].func(t).subs(t, 0)
+            else:
+                y = Derivative(ma1[g].func(t), (t, k)).subs(t, 0)
+            d.append(s**(ma1[n]-k-1)*y)
+        r = s**ma1[n]*_laplace_apply_rules(ma1[g].func(t), t, s, **hints)
+        return r - Add(*d)
+    raise IntegralTransformError(
+                'Laplace', None, 'No rules could be applied.')
```

Comment:
Maybe this could just return None?

Reply:
An then write the part 

https://github.com/sympy/sympy/blob/b5191b805eba30488a54cc0e46dbab66052a0429/sympy/integrals/transforms.py#L1711-L1719

as 

```
L = _laplace_apply_rules(f, t, s, **hints)
if L==None:
    return _laplace_transform(f, t, s, **hints)
else:
    return L
```

?

Reply:
Yes

Reply:
OK!